commands/narrator/srole.js

```diff
@@ -207,11 +207,10 @@ module.exports = {
         shuffle(finalRoleList)
         let sorcChats = []
         for (let k = 0; k < alive.members.size; k++) {
-            let theirRole = finalRoleList[k]
+            let theirRole = finalRoleList[k].name
             let role = getRole(theirRole)
             rolelist.push(theirRole)
             let guy = message.guild.members.cache.find((x) => x.nickname == `${k + 1}`)
-            db.delete(`suicided_${guy.id}`)
```

Comment:
Again, this line doesn't need to be deleted

Reply:
Hold up i deleted that?